src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -700,8 +700,13 @@ Copyright (c) .NET Foundation. All rights reserved.
                                        Condition="'$(PublishReferencesDocumentationFiles)' == 'true' or '%(ReferenceCopyLocalPaths.Extension)' != '.xml'">
         <DestinationSubPath>%(ReferenceCopyLocalPaths.DestinationSubDirectory)%(ReferenceCopyLocalPaths.Filename)%(ReferenceCopyLocalPaths.Extension)</DestinationSubPath>
       </_ResolvedCopyLocalPublishAssets>
-    </ItemGroup>
 
+      <!-- Include RuntimePackAsset where RuntimePackAlwaysCopyLocal is true -->
+      <_ResolvedCopyLocalPublishAssets Include="@(RuntimePackAsset)"
+                              Condition="'%(RuntimePackAsset.RuntimePackAlwaysCopyLocal)' == 'true'">
```

Comment:
Is this guaranteed to always be true for `Microsoft.Windows.SDK.NET`?

Reply:
yes. This is a flag to tell Microsoft.Windows.SDK.NET apart from other runtime assets that installed on client machine, therefor need to copylocal